### PR TITLE
An example solution

### PR DESCRIPTION
**Pull request by silvabox**

### CODE REVIEW

File: lib/menu.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  include Enumerable
+  extend Forwardable
+
+  def_delegators :items, :each, :include?, :member?, :empty?
+
+  def self.from_hash(hash)
+    menu = Menu.new
+    hash.each { |name, price| menu.add(name, price) }
+    menu
+  end
```
Comment (ptolemybarnes) [inline lib/menu.rb]: Could be:
```
def self.from_hash(hash)
   hash.each_with_object(Menu.new) do |(name, price), menu|
      menu.add(name, price)
   end
 end
```

If ya wanna be cool

File: lib/menu.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  def add(name, price)
+    items[name] = price
+  end
+
+  def price_of(name)
+    items[name]
+  end
+
+  def to_s
+    string = ""
```
Comment (pitchinvasion) [inline lib/menu.rb]: `each_with_object` works here too

File: spec/menu_spec.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+require 'menu'
+
+describe Menu do
+  subject(:menu) { Menu.new }
+  let(:items) { { :'Pizza' => 9.25, :'Chips' => 2.00 } }
```
Comment (sjmog) [inline spec/menu_spec.rb]: Does it not make more sense to use lowercase symbols and some sort of `humanize` method in a view layer?

Comment (silvabox) [inline spec/menu_spec.rb]: Actually, in production they wouldn't be symbols at all - I should probably just use strings here.

File: lib/menu.rb (hunk 1 above)
Comment (silvabox) [inline lib/menu.rb]: Like!  I didn't know that!

**Pull request closed without merging**